charts/kube-prometheus-stack/templates/prometheus-operator/clusterrole.yaml

```diff
@@ -11,14 +11,19 @@ rules:
   - monitoring.coreos.com
   resources:
```

Comment:
@QuentinBisson is it possible to generate this list? Or its an manual doing?

Reply:
It is done manually at the moment but we can probably have it done automatically as it's coming from https://github.com/prometheus-operator/kube-prometheus/blob/3fff8b56097778f1491edba968aee3cfbd5652ef/manifests/prometheusOperator-clusterRole.yaml#L13